Dockerfile

```diff
@@ -40,6 +40,6 @@ RUN mkdir -p log tmp bin
 
 RUN adduser rails -D -h /rails -s /bin/sh && \
   chown -R rails:rails db log tmp bin && \
-  chmod 755 /rails/bin/docker-entrypoint
+  chmod 755 /app/bin/docker-entrypoint
 
 USER rails:rails
```

Comment:
`rails` was not causing any problems but it made it a bit more difficult to grock these different scripts.